test/system/post_test.rb

```diff
@@ -0,0 +1,32 @@
+require "application_system_test_case"
+# https://guides.rubyonrails.org/testing.html#implementing-a-system-test
+
+class PostTest < ApplicationSystemTestCase
+
+  def setup
+    activate_authlogic
+  end
+
+  test 'posting from the editor' do
+    visit '/'
+
+    click_on 'Login'
+    fill_in("username-login", with: "Bob")
+    fill_in("password-signup", with: "secretive")
+    click_on "Log in"
```

Comment:
You are logging in first and then going to '/post`. I think it should be go to the '/post` being not logged in then the person is redirected to the '/login' page then person logs in.....

What do you suggest Jeff?

Reply:
hmmm, good idea... let me try...